yellow-paper/docs/contract-deployment/classes.md

```diff
@@ -124,13 +126,16 @@ In pseudocode:
 function register(
   artifact_hash: Field,
   private_functions_root: Field,
+  public_bytecode_commitment: Field,
```

Comment:
This will need to be an encoding of a point, so at least 1 Field + 1 bit.

Reply:
Can't we use the "all points are represented on one side of the curve" to avoid that extra bit?

Reply:
I don't know 😅 Potentially. We'd need to check with the crypto team, I think.
A follow-up thought that just popped into my mind. The bytecode commitment might be an altBN254 point, rather than a Grumpkin point, so the altBN254 coordinates would be (F_q, F_q) instead of Grumpkin coordinates (F_r, F_r). I believe q > r, so even a single coordinate wouldn't fit into an F_r field. (The `Field` type in Noir is F_r).

Reply:
Redefining it to be a Point for now, until we know for sure what shape it'll have